zk/tests/conftest.py

```diff
@@ -117,8 +115,14 @@ def condition():
                 time.sleep(1)
 
         if not booted:
-            raise RetryError("Zookeeper failed to boot!")
+            # raise RetryError("Zookeeper failed to boot!")
+            return True
```

Comment:
Why returning `True` here ? I don't think we want to continue if the container failed to load

Reply:
oups forgot this! my bad